packages/flutter/test/widgets/transitions_test.dart

```diff
@@ -283,18 +289,139 @@ void main() {
 
     final RenderPositionedBox actualPositionedBox = tester.renderObject(find.byType(Align));
     expect(actualPositionedBox.widthFactor, 0.0);
+    expect(actualPositionedBox.heightFactor, 1.0);
 
     controller.value = 0.0;
     await tester.pump();
     expect(actualPositionedBox.widthFactor, 0.0);
+    expect(actualPositionedBox.heightFactor, 1.0);
 
     controller.value = 0.75;
     await tester.pump();
     expect(actualPositionedBox.widthFactor, 0.5);
+    expect(actualPositionedBox.heightFactor, 1.0);
+
+    controller.value = 1.0;
+    await tester.pump();
+    expect(actualPositionedBox.widthFactor, 1.0);
+    expect(actualPositionedBox.heightFactor, 1.0);
+  });
+
+  testWidgetsWithLeakTracking('SizeTransition with crossAxisSizeFactor should size its cross axi from its children - vertical axis', (WidgetTester tester) async {
```

Comment:
`axi` -> `axis`. Same for horizontal test.

Reply:
Oops sorry about that. I fixed it in [doc: Fix typo in test description](https://github.com/flutter/flutter/pull/134659/commits/0bcb62840461a02fd79d1da120c43d199893b1f0)